clients/src/main/java/org/apache/kafka/common/protocol/CommonFields.java

```diff
@@ -26,6 +26,11 @@
     public static final Field.Int32 PARTITION_ID = new Field.Int32("partition", "Topic partition id");
     public static final Field.Int16 ERROR_CODE = new Field.Int16("error_code", "Response error code");
     public static final Field.NullableStr ERROR_MESSAGE = new Field.NullableStr("error_message", "Response error message");
+    public static final Field.Boolean VALIDATE_ONLY = new Field.Boolean("validate_only",
+            "If this is true, the request will be validated, but the " +
+                     "topic won't be created.");
```

Comment:
Seems like "topic won't be created" is specific to one of the requests.

Reply:
Fixed.